src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1266,4 +1266,22 @@
   <data name="DataSetReadXmlAutogeneratedTitle" xml:space="preserve">
     <value>Ensure autogenerated class containing DataSet.ReadXml() is not used with untrusted data</value>
   </data>
+  <data name="DoNotDisableTokenValidationChecks" xml:space="preserve">
+    <value>Do not disable token validation checks</value>
```

Comment:
nit: put period at the end of the sentence for consistency

Reply:
Don't need punctuation for titles (and probably should rename the string to end with "Title" like @Evangelink suggested)